lib/rules/typescript-uses-vars.js

```diff
@@ -0,0 +1,154 @@
+/**
```

Comment:
I still think `no-unused-vars` makes more sense because it will trigger warnings from the core rule of the same name.